### ISSUE TITLE
Support FaceSplit Splits

### ISSUE THREAD

**Issue by CryZe** [glacials/splits-io#198]
https://github.com/nolife9face/FaceSplit

Splits are formatted like this:

```
Title
Goal
25
2
1 Death-0-0-1,19-
2 Deaths"?"-0-0-1,88-
3 Deaths-0-0-0,52-
4 Deaths-0-0-0,1-D:\Dokumente\WSplit\Icons\Ocarina of Time\PNG\Blue Potion.png
5 Deaths-3,28-0-0,14-D:\Dokumente\WSplit\Icons\Ocarina of Time\PNG\Biggoron's Sword.png
6 Deaths-3,89-0,6-0,6-D:\Dokumente\WSplit\Icons\Ocarina of Time\PNG\Big Poe.png
7 Deaths-0-0-0,21-D:\Dokumente\WSplit\Icons\Ocarina of Time\PNG\Dash-Dash.png
8 Deaths-0-0-0,32-D:\Dokumente\WSplit\Icons\Ocarina of Time\PNG\Boomerang.png
9 Deaths-5,34-0-0,4-
10 Deaths-6,3-0,95-0,15-
```

The first 4 lines are Title, Goal, Attempt Count, Finished Attempts and then there's one line per segment, which uses a dash as the separator of the values stored in there: Segment Name, Split Time, Segment Time, Best Segment Time, Icon Path. The last dash and Icon Paths are only in 1.1 splits and not earlier versions. Also dashes in split names are escaped as `"?"`. Times are in seconds and have either a `,` or a `.` as a decimal separator.

**Comment by glacials** [comment]
Thanks for spelling this out :D will add when I get the chance.

**Comment by BatedUrGonnaDie** [comment]
Closed with #312
